CONTRIBUTING.md

```diff
@@ -321,3 +321,42 @@ stripped with no prior warning or consent from the member in question.
 Earning this privilege should be considered to be no small feat and is by no
 means guaranteed by any quantifiable metric. It is a symbol of great trust of
 the community of this project.
+
+## Point Release Procedure
```

Comment:
This says one thing, yet the rest of document seems to focus on things that are not necessarily related to release management, e.g. ADRs

Reply:
all of the document relates to contributing though. Where else would this information live?

Reply:
They should definitely be in `CONTRIBUTING.md`, just in a dedicated stanza maybe?